server/app/src/test/java/com/google/heartrate/wearos/app/gatt/heartrate/characteristics/SetHeartRateCharacteristicValueInRangeTest.java

```diff
@@ -0,0 +1,49 @@
+package com.google.heartrate.wearos.app.gatt.heartrate.characteristics;
+
+import android.os.Build;
+
+import com.google.heartrate.wearos.app.gatt.CharacteristicsArgumentProvider;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.ParameterizedRobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.util.Collection;
+
+import static com.google.heartrate.wearos.app.gatt.CharacteristicsArgumentProvider.Range.IN_UINT16;
+
+@RunWith(ParameterizedRobolectricTestRunner.class)
+@Config(sdk = {Build.VERSION_CODES.O_MR1}, manifest = Config.NONE)
+public class SetHeartRateCharacteristicValueInRangeTest {
```

Comment:
Would be nice if this could be folded into the HeartRateGatt test above.

Reply:
I don't understand, you mean moving this test into HeartRateGatt folder or move to one class? I can only put into one  folder, because I can not put classes `SetExpendedEnergyValueOutOfRangeTest`, `SetHeartRateCharacteristicValueInRangeTest`, `SetHeartRateCharacteristicValueOutOfRangeTest`, `SetHeartRateMeasurementValueOutOfRangeTest` tests in one class. `ParameterizedRobolectricTestRunner` requires only one `@ParameterizedRobolectricTestRunner.Parameters` (method which generates values) in each test class(((

Reply:
That's exactly what I wanted to double check. Thanks!